src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -842,6 +875,7 @@ private async Task CommitCoreAsync(IUIThreadOperationContext operationContext, b
                 if (newSolution == null)
                 {
                     // User clicked cancel.
+                    CommitStateChange?.Invoke(this, false);
```

Comment:
seems error prine to have to issue these.  can we have this be in a finally somewhere?

Reply:
Yeah I have put it into finally cause>